src/PrettyPrinterTrait.php

```diff
@@ -54,7 +58,9 @@ public function endTest(Test $test, float $time): void
         $name = preg_replace('/^test /', '', $name, 1);
 
         // Get the data set name
-        $name = $this->handleDataSetName($name, $testMethodName[1]);
+        if ($dataSet) {
+            $name .= ' [ ' . preg_replace('/^"|"$/', '', $dataSet) . ' ]';
```

Comment:
```suggestion
            // Note: Use preg_replace() instead of trim() because the dataset may end with a quote
            // (double quotes) and trim() would remove both from the end. This matches only a single
            // quote from the beginning and end of the dataset that was added by PHPUnit itself.
            $name .= ' [ ' . preg_replace('/^"|"$/', '', $dataSet) . ' ]';
```